includes/class-freemius.php

```diff
@@ -4595,13 +4763,49 @@ private static function get_all_sites() {
 		}
 
 		/**
+		 * @author Leo Fajardo (@leorw)
+		 *
+		 * @since 1.2.1
+		 *
+		 * @param string $option_name
+		 * @param string $module_type
+		 */
+		private static function get_account_option( $option_name, $module_type ) {
+			if ( self::MODULE_TYPE_PLUGIN !== $module_type ) {
+				$option_name = $module_type . '_' . $option_name;
+			}
+
+			return self::$_accounts->get_option( $option_name, array() );
+		}
+
+		/**
+		 * @author Leo Fajardo (@leorw)
+		 *
+		 * @since 1.2.1
+		 *
+		 * @param string $option_name
+		 * @param mixed  $option_value
+		 * @param bool   $store
+		 * @param number $module_id
+		 */
+		private function set_account_option( $option_name, $option_value, $store ) {
```

Comment:
@vovafeldman `get_account_option()` is only called in static methods. Two of those caller methods (`get_all_sites` and `get_all_licenses`) are called in `_debug_page_render` method that doesn't know about the module ID. If I will change `get_account_option()` to a non-static method, then I guess I will also have to change the mentioned caller methods so that they will receive the module ID instead of the module type? Then I can get the instance from the module ID. After that, I will also have to modify `_debug_page_render` and get it to load all sites and licenses using a new method or using `self::$_accounts` directly.